core/web/user_controller.go

```diff
@@ -170,7 +170,7 @@ func getETHAccount(ctx *gin.Context, store *store.Store, account accounts.Accoun
 		return presenters.ETHKey{}
 	}
 
-	linkBalance, err := txm.GetLINKBalance(account.Address)
+	linkBalance, err := bulletprooftxmanager.GetLINKBalance(store.Config, store.EthClient, account.Address)
```

Comment:
Would it do better to move this method onto the EthClient?

Reply:
I think that's definitely a better place for this. Thanks.